### PR TITLE
add nfs provisioner as plugin

### PR DESCRIPTION
**Pull request by g-iannelli**

### CODE REVIEW

**Review by nutellinoit — DISMISSED** [review]
LGTM!

File: katalog/nfs-subdir-external-provisioner/MAINTENANCE.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# `nfs-subdir-external-provisioner` Package Maintenance
+
+To prepare a new release of this package:
+
+1. Generate manifests from helm charts
+```bash
+helm repo add nfs-subdir-external-provisioner https://kubernetes-sigs.github.io/nfs-subdir-external-provisioner
+helm template --release-name nfs-subdir-external-provisioner nfs-subdir-external-provisioner/nfs-subdir-external-provisioner --namespace nfs-subdir-external-provisioner | tee deploy.yaml
```
Comment (ralgozino) [inline katalog/nfs-subdir-external-provisioner/MAINTENANCE.md:8]: can we write down here the version of the chart that you used? so the next maintainer knows where to start from.

File: katalog/nfs-subdir-external-provisioner/MAINTENANCE.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+   fi
+done < "deploy.yaml"
+ 
+# Save the last processed file
+if [ -n "$current_file" ]; then
+  echo -e "$sighup_copyright\n$current_content" > "resources/$current_file"
+fi
+ 
+EOF
+```
```
Comment (ralgozino) [inline katalog/nfs-subdir-external-provisioner/MAINTENANCE.md:42]: is this logic here to add the license header?

you can do this with `addlicense` (the same binary we use in CI to check that the header is present):

```bash
addlicense -c "SIGHUP s.r.l" -v -l bsd -y "2017-present" ./resources```

**Review by ralgozino — CHANGES_REQUESTED** [review]
Two very minor observations. Great job!

File: katalog/nfs-subdir-external-provisioner/MAINTENANCE.md (hunk 1 above)
Comment (g-iannelli) [reply katalog/nfs-subdir-external-provisioner/MAINTENANCE.md]: I edit the file specifing the current `nfs-subdir-external-provisioner` helm chart version.
Morover I moved the bash in the file `MAINTENANCE.sh`

**Review by ralgozino — APPROVED** [review]
lgtm!

**Review by ralgozino — APPROVED** [review]

**Pull request merged**